cfgov/jinja2/v1/wagtail/molecules/image-text-25-75.html

```diff
@@ -0,0 +1,27 @@
+<div class="m-image-text-25-75">
+
+    {% if has_rule %}
+        <div class="a-rule-break"></div>
```

Comment:
It's a horizontal rule. It's mostly used for decoration now, because `section` has been implemented.